repose-aggregator/tests/performance-tests/roles/gatling/files/gatling.conf

```diff
@@ -0,0 +1,132 @@
+    #mute = false                 # When set to true, don't ask for simulation name nor run description (currently only used by Gatling SBT plugin)
+    #elFileBodiesCacheMaxCapacity = 200        # Cache size for request body EL templates, set to 0 to disable
+    #rawFileBodiesCacheMaxCapacity = 200       # Cache size for request body Raw templates, set to 0 to disable
+    #rawFileBodiesInMemoryMaxSize = 1000       # Below this limit, raw file bodies will be cached in memory
+
+    extract {
+      regex {
+        #cacheMaxCapacity = 200 # Cache size for the compiled regexes, set to 0 to disable caching
+      }
+      xpath {
+        #cacheMaxCapacity = 200 # Cache size for the compiled XPath queries,  set to 0 to disable caching
+      }
+      jsonPath {
+        #cacheMaxCapacity = 200 # Cache size for the compiled jsonPath queries, set to 0 to disable caching
+        #preferJackson = false  # When set to true, prefer Jackson over Boon for JSON-related operations
+      }
+      css {
+        #cacheMaxCapacity = 200 # Cache size for the compiled CSS selectors queries,  set to 0 to disable caching
+      }
+    }
+
+    directory {
+      #data = user-files/data               # Folder where user's data (e.g. files used by Feeders) is located
+      #bodies = user-files/bodies           # Folder where bodies are located
+      #simulations = user-files/simulations # Folder where the bundle's simulations are located
+      #reportsOnly = ""                     # If set, name of report folder to look for in order to generate its report
+      #binaries = ""                        # If set, name of the folder where compiles classes are located: Defaults to GATLING_HOME/target.
+      #results = results                    # Name of the folder where all reports folder are located
+    }
+  }
+  charting {
+    #noReports = false       # When set to true, don't generate HTML reports
+    #maxPlotPerSeries = 1000 # Number of points per graph in Gatling reports
+    #useGroupDurationMetric = false  # Switch group timings from cumulated response time to group duration.
+    indicators {
+      #lowerBound = 800      # Lower bound for the requests' response time to track in the reports and the console summary
+      #higherBound = 1200    # Higher bound for the requests' response time to track in the reports and the console summary
+      percentile1 = 50      # Value for the 1st percentile to track in the reports, the console summary and Graphite
+      percentile2 = 95      # Value for the 2nd percentile to track in the reports, the console summary and Graphite
+      percentile3 = 99      # Value for the 3rd percentile to track in the reports, the console summary and Graphite
+      percentile4 = 99.9    # Value for the 4th percentile to track in the reports, the console summary and Graphite
```

Comment:
This might exacerbate a bug we have in the Ansible stuff.

https://github.com/rackerlabs/repose/blob/master/repose-aggregator/tests/performance-tests/roles/gatling/defaults/main.yml#L18

Even my TO DO comment is misleading. What it really does is look at the `percentile4` value.

https://github.com/rackerlabs/repose/blob/master/repose-aggregator/tests/performance-tests/roles/gatling/files/simulations/filters/saml/SamlSimulation.scala#L120

At the very least, I recommend changing the `500` value to something larger, maybe `3000`. The big reason is because if it fails to meet this, the `gatling` command returns a non-negative status code which causes the Ansible run to abruptly stop.

Slightly better solution would be to change all of the simulations to use `percentile3` instead.

Best would include fixing the name of the Ansible variable. I never did because I wasn't 100% sure what the point of it was. My initial guess was it would be a red flag in the HTML report if the expectation wasn't met, but in reality, it seems to be treated as a "if this isn't met, something terrible must have happened and the results are useless, so throw them away."